README.md

```diff
@@ -28,11 +28,12 @@ To perform a release simply do: `mvn clean deploy -DperformRelease=true`
 ## Release Notes
 
 ### 0.1.6
-Added attributes for gravitational multigraph and multigraph aggregation pipeline
```

Comment:
I think we need to have a full documentation on what a math expression is. I for sure have no idea what that is :)